pkg/web/routes.go

```diff
@@ -148,6 +149,7 @@ func (s *Server) setupRoutes() (err error) {
 			transactions.PUT("/:id", authorize(permiss.TravelRuleManage), s.UpdateTransaction)
 			transactions.DELETE("/:id", authorize(permiss.TravelRuleDelete), s.DeleteTransaction)
 			transactions.GET("/:id/preview", authorize(permiss.TravelRuleManage), s.AcceptTransactionPreview)
+			transactions.GET("/:id/info", authorize(permiss.TravelRuleView), s.TransactionInfo)
```

Comment:
This endpoint displays the transaction detail preview. Since we have an endpoint with `/preview` using `/info` felt best as an alternative.